visual_servo:
  ros__parameters:
    target_class: 'apple'
    kp_x: 0.5
    kp_y: 0.5
    ki_x: 0.01
    ki_y: 0.01
    kd_x: 0.1
    kd_y: 0.1
    control_rate: 50.0
    image_width: 1280
    image_height: 720
    target_tolerance: 10

depth_estimator:
  ros__parameters:
    depth_topic: '/camera/depth'
    camera_info_topic: '/camera/camera_info'
    base_frame: 'base_link'
    camera_frame: 'camera_optical_frame'
